sycl/test-e2e/DeviceImageBackendContent/CUDA_interop_test.cpp

```diff
@@ -0,0 +1,33 @@
+// RUN: %{build} %cuda_options -o %t.out
+// RUN: %{run} %t.out
+// REQUIRES: cuda, cuda_dev_kit
+
+#include <cuda.h>
+#include <sycl/backend.hpp>
+#include <sycl/detail/core.hpp>
+#include <vector>
+
+int main() {
+  sycl::device d([](const sycl::device &d) {
+    return d.get_backend() == sycl::backend::opencl;
```

Comment:
Why does CUDA interop test requires OpenCL backend?

Reply:
That's definitely a mistake, thanks for the catch!